packages/material-ui/src/NativeSelect/NativeSelectInput.js

```diff
@@ -7,18 +7,13 @@ import capitalize from '../utils/capitalize';
 import nativeSelectClasses, { getNativeSelectUtilitiyClasses } from './nativeSelectClasses';
 import experimentalStyled from '../styles/experimentalStyled';
 
-export const overridesResolver = (props, styles) => {
+const overridesResolver = (props, styles) => {
   const { styleProps } = props;
 
   return deepmerge(
     {
       ...styles.select,
       ...styles[styleProps.variant],
-      [`& .${nativeSelectClasses.icon}`]: {
```

Comment:
The icon is rendered next to the root, so we cannot overrides with this nested selector.